File: Proyectos/ComparacionJava/CompararObjetosRedefiniendoEquals/src/CompararObjetosRedefiniendoEquals/Direccion.java

package CompararObjetosRedefiniendoEquals; 

 
 class Direccion implements Cloneable{
      String calle;
      int numero;
      String codigoPostal;
      String ciudad;

    Direccion(String calle, int num, String cp, String ciudad) {
        this.calle = calle;
        this.numero = num;
        this.codigoPostal = cp;
        this.ciudad = ciudad;    
    } 
     
 
}

File: Proyectos/comparacion_ruby/comparacion_objetos_inicial/lib/direccion.rb


class Direccion
  attr_reader :calle, :numero, :codigoPostal
  attr_accessor :ciudad
  def initialize(calle, numero, cp, ciudad)
    @calle = calle
    @numero = numero
    @codigoPostal = cp
    @ciudad = ciudad    
  end
 
end
